contracts/test/echidna/TierReport_Echidna.sol

```diff
@@ -0,0 +1,288 @@
+// SPDX-License-Identifier: CAL
+pragma solidity =0.8.10;
+
+import {TierReport} from "../../tier/libraries/TierReport.sol";
+import {TierConstants} from "../../tier/libraries/TierConstants.sol";
+
+/// @title TierReport_Echidna
+/// Wrapper around the `TierReport` library for echidna fuzz testing.
+contract TierReport_Echidna {
```

Comment:
can we get some instructions in the readme for how to run this?

should it run on CI?

Reply:
I added it, sorry. I'm working on other branch from this. Could you review again and merge it if all good? So I can work from develop again :)